api/src/controllers/events/click-event.ts

```diff
@@ -29,4 +29,22 @@ const clickEventRoute = APIWrapper({
     },
 });
 
-export const clickEvent = relogRequestHandler(clickEventRoute);
\ No newline at end of file
+const clickEventPaginationRoute = APIWrapper({
+    GET: {
+        config: {
+            requireToken: true,
+        },
+        handler: async (req: Request) => {
+            const { afterTime, afterId, limit, projectName } = req.params;
+
+            if (!afterId || !afterTime || !limit || !projectName) {
```

Comment:
if afterTime is undefined, default it to 30 days from the current time instead of throwing an error